tests/ens/test_offchain_resolution.py

```diff
@@ -173,14 +173,14 @@ def test_offchain_resolver_function_call_raises_with_ccip_read_disabled(
     with pytest.raises(OffchainLookup):
         offchain_resolver.functions.resolve(
             ens_encode_name("offchainexample.eth"),
-            ENCODED_ADDR_CALLDATA,
+            ENCODED_ADDR_CALLDATA.encode(),
```

Comment:
Why the added `encode` here? Test passes for me with or without it.

Reply:
No need for this after all, good callout.